tests/test_datapackage.py

```diff
@@ -345,6 +345,44 @@ def test_raises_if_remote_resource_url_doesnt_exist(self):
         with pytest.raises(IOError):
             datapackage.DataPackage(descriptor).resources[0].data
 
+    @httpretty.activate
+    def test_if_relative_resource_of_remote_package_has_absolute_path(self):
```

Comment:
We should be fine considering that "no local data path" means `local_data_path() is None`. So please delete this test :+1: